file-service-it/src/test/java/uk/gov/justice/services/fileservice/repository/ContentJdbcRepositoryIT.java

```diff
@@ -60,13 +59,12 @@ public void shouldStoreAndRetrieveFileContent() throws Exception {
         final InputStream content = new ByteArrayInputStream(contentString.getBytes());
         contentJdbcRepository.insert(fileId, content, connection);
 
-        final Optional<InputStream> fileContents = contentJdbcRepository.findByFileId(fileId, connection);
+        final FileContent fileContent = contentJdbcRepository
+                .findByFileId(fileId, connection)
+                .orElseThrow(() -> new AssertionError("Failed to find file contents"));
 
-        assertThat(fileContents.isPresent(), is(true));
-
-        final InputStream contentStream = fileContents.orElseThrow(() -> new AssertionError("Failed to find file contents"));
-
-        assertThat(IOUtils.toString(contentStream), is(contentString));
+        assertThat(IOUtils.toString(fileContent.getContent()), is(contentString));
```

Comment:
No - unfortunately not. It would result in just toString() which conflicts with the default toString() on every object